dask/dataframe/io/tests/test_parquet.py

```diff
@@ -125,14 +125,14 @@ def test_roundtrip_from_pandas():
 
 def test_categorical():
     with tmpdir() as tmp:
-        df = pd.DataFrame({'x': ['a', 'b', 'c'] * 100},
+        df = pd.DataFrame({'x': [u'a', u'b', u'c'] * 100},
```

Comment:
I'd rather not, because that would also affect, for example, the column name 'x' in this line, and probably other places in the code.